_sass/layout.scss

```diff
@@ -27,7 +27,8 @@
 	}
 
 	.notMobile {
-		display: none !important;
+		// display: none !important;
```

Comment:
I don't like commented out code, if we need old code we can go back to older commits :)

```suggestion
```